src/CppUTestExt/IEEE754ExceptionsPlugin.cpp

```diff
@@ -71,7 +71,7 @@ void IEEE754ExceptionsPlugin::enableInexact()
 
 bool IEEE754ExceptionsPlugin::checkIeee754ExeptionFlag(int flag)
 {
-    return fetestexcept(flag);
+    return !! fetestexcept(flag);
```

Comment:
Ok can either you make that pull request or @asgeroverby change the pull request